sites/public/pages/index.tsx

```diff
@@ -177,15 +181,7 @@ export default function Home({ latestListings }) {
         className={styles["latest-listings"]}
       >
         {latestListings.items.map((listing) => {
-          return (
-            <LatestListingsLink
-              key={listing.id}
-              name={listing.name}
-              address={listing.buildingAddress}
-              availability={buildUnitSummaryString(listing.unitsSummary)}
-              imageUrl={imageUrlFromListing(listing, 520)}
-            />
-          )
+          return <LatestListingsLink key={listing.id} listing={listing} />
```

Comment:
Is the `key` prop needed? Seems like LatestListingsLink only uses `props.listing`.

Reply:
Yeah, it's a special react prop for when you are generating items from a list: https://reactjs.org/docs/lists-and-keys.html#keys

Basically it complained when I didn't have this